src/main/java/com/google/api/codegen/gapic/GapicGeneratorApp.java

```diff
@@ -112,6 +113,14 @@
           "The filepath of the JSON gRPC Service Config file.",
           "");
 
+  public static final Option<String> TRANSPORT =
+      ToolOptions.createOption(
+          String.class,
+          "transport",
+          "List of transports to use ('rest' or 'grpc') separated by '+'. NOTE: For now"
```

Comment:
Should this just be "Transport to use" since it's not a List option?

Reply:
Again, change the NOTE to be accurate: for now, we only support a single transport.